specifyweb/frontend/js_src/lib/components/Interactions/LoanReturn.tsx

```diff
@@ -205,6 +235,36 @@ function PreparationReturn({
           resource={loanReturnPreparation.current}
           viewDefinition={loanReturnPrepForm()}
         />
+        <div className="flex justify-end gap-2">
```

Comment:
since fields are inline, not on top of each other, I would suggest adding more gap to this `<div>` to better differentiate the fields (i.e change gap-2 to gap-4 or gap-8)

also, just because the person who sent you the screenshot had these fields right-aligned, doesn't mean you should implement them that way - decide what do you think looks the best or produces best UX